packages/frontend/src/pages/SqlRunnerNew.tsx

```diff
@@ -16,63 +18,67 @@ const SqlRunnerNewPage = () => {
     const [isRightSidebarOpen, setRightSidebarOpen] = useState(false);
 
     return (
-        <Page
-            title="SQL Runner"
-            noContentPadding
-            flexContent
-            header={<Header />}
-            isSidebarOpen={isLeftSidebarOpen}
-            sidebar={
-                <Sidebar
-                    projectUuid={selectedProjectUuid}
-                    setSidebarOpen={setLeftSidebarOpen}
-                />
-            }
-            isRightSidebarOpen={isRightSidebarOpen}
-            rightSidebar={<RightSidebar setSidebarOpen={setRightSidebarOpen} />}
-        >
-            <Group
-                align={'stretch'}
-                grow
-                spacing="none"
-                p={0}
-                style={{ flex: 1 }}
+        <Provider store={store}>
```

Comment:
Can we require the project uuid to be passed here? + remove prop drilling

Reply:
oooh checking!

Reply:
LMK what you think of this approach: https://github.com/lightdash/lightdash/pull/10589/commits/27eb69d1da2595d1fe22f1a5106a12b140b534c2